frontends/bmd/src/pages/admin_screen.tsx

```diff
@@ -84,41 +74,24 @@ export function AdminScreen({
         <Prose>
           {election && (
             <React.Fragment>
-              <h1>
-                <label htmlFor="selectPrecinct">Precinct</label>
-              </h1>
-              <p>
-                <Select
-                  id="selectPrecinct"
-                  value={
-                    appPrecinct?.kind === 'AllPrecincts'
-                      ? ALL_PRECINCTS_OPTION_VALUE
-                      : appPrecinct?.precinctId ?? ''
-                  }
-                  onBlur={changeAppPrecinctId}
-                  onChange={changeAppPrecinctId}
-                >
-                  <option value="" disabled>
-                    Select a precinct for this device…
-                  </option>
-                  {election.precincts.length > 1 && (
-                    <option value={ALL_PRECINCTS_OPTION_VALUE}>
-                      {ALL_PRECINCTS_NAME}
-                    </option>
-                  )}
-                  {[...election.precincts]
-                    .sort((a, b) =>
-                      a.name.localeCompare(b.name, undefined, {
-                        ignorePunctuation: true,
-                      })
-                    )
-                    .map((precinct) => (
-                      <option key={precinct.id} value={precinct.id}>
-                        {precinct.name}
-                      </option>
-                    ))}
-                </Select>
-              </p>
+              {election.precincts.length > 1 && (
+                <React.Fragment>
+                  <h1>
+                    <label htmlFor="selectPrecinct">Precinct</label>
+                  </h1>
+                  <ChangePrecinctButton
+                    appPrecinctSelection={appPrecinct}
+                    updatePrecinctSelection={makeAsync(updateAppPrecinct)}
+                    election={election}
+                    mode={
+                      pollsState === 'polls_closed_final'
+                        ? 'disabled'
+                        : 'default'
+                    }
+                    logger={logger}
+                  />
+                </React.Fragment>
+              )}
```

Comment:
I forgot to add a screenshot for this in the description, but I made it so that we just hide the "Precinct" section of the admin screen if there's only one precinct and nothing to change.